stdgo/reflect/Reflect.hx

```diff
@@ -484,6 +496,27 @@ class Type {
 		return 0;
 	}
 
+	public function method(index:GoInt):Method {
+		switch gt {
+			case GT_namedType(_, _, methods, _, _, _), GT_interface(_, _, methods):
+				var method = methods[index.toBasic()];
+				switch method {
+					case GT_field(name, type, tag):
+						return {
+							name: name,
+							pkgPath: "",
+							type: new Type(type),
+							func: null,
```

Comment:
Eventually this will need to be a closure that calls the method using Haxe reflection of the path+name.